service_account_keys.js

```diff
@@ -0,0 +1,34 @@
+const helpers = require('./helpers');
+const apiAdapter = require('./api_adapter');
+
+function maybeTalkToBond(auth) {
+    return apiAdapter.getJsonFrom(
+        `${helpers.bondBaseUrl()}/api/link/v1/fence/serviceaccount/key`,
+        auth
+    ).catch((e) => {
+        console.error('Unable to retrieve Service Account Key from Bond');
+        throw e;
+    });
+}
+
+function maybeTalkToSam(auth) {
+    return apiAdapter.getJsonFrom(
+        `${helpers.samBaseUrl()}/api/google/v1/user/petServiceAccount/key`,
```

Comment:
we have had problems with the service account returned here not being able to call the storage json api... we should test early that the signed URL returned using this SA works the way we want

Reply:
I can verify that this SA key can't get json metadata (I tried, it will only get xml), but can still get a signed url.

Reply:
getting a signed url is not the same as using a signed url. Anybody can sign a URL, even without access. We know the SA can't get json metadata. Is there the same problem using the singed url.

Reply:
Ah, I have tried the signed url's it generates and they have all worked thus far, but do you mean we should validate programmatically? If so, how?

Reply:
that's all, I didn't mean anything fancy